### PR TITLE
The crashed abductor ship is now fully deconstructable

### LINKED ISSUE #37741: Crashed Abductor Ship Walls Unbreakable
So I played as a Free Golem recently and was attempting to salvage parts from the Abductor ship to make abductor golems.  Turns out the walls cannot be broken down via welder or Gibonite explosions.  I asked an admin about this and he went code diving and discovered that the walls **should** be breakable, but the tiles for some reason were set to not be destructible.  This sounded like a bug, so I'm posting it here.

I still managed to salvage Alien Alloy from the tables, lockers, and surrounding equipment.

### PR DESCRIPTION
**Pull request by ghost**
:cl:
tweak: the walls on the crashed abductor ship ruin in lavaland can now be broken down
/:cl:

fixes #37741

### CODE REVIEW

**Comment by Repukan** [comment]
Oh goody now golem players can metagame a path to the ship so they can be the strongest golem race with no repercussions

**Comment by fludd12** [comment]
>implying they couldn't before?

**Comment by Repukan** [comment]
They could but they barely had enough materials to scrape two shells at most.

**Comment by fludd12** [comment]
Except golems get r&d, so they can just scan the tools and churn out more alien metal because it's an alloy of literally the two most common materials

**Comment by Repukan** [comment]
Yes, but that requires more effort and time than using your pre spawned toolbox to make a beeline to the ruin and just deconstructing it.

**Comment by fludd12** [comment]
But, since you can already do that with the other stuff, it's easy to make your own body alien, then use your advanced speed to get r&d and do everyone else anyway - this PR changes almost nothing other than removing a spot where you can corner a megafauna and meme them before diving behind unbreakable cover.

**Comment by iskyp** [comment]
>abductor golems are the most overpowered
>not runic metal golems

**Comment by ma44** [comment]
>Doesn't have a GPS

**Comment by timothymtorres** [comment]
The amount of alien alloy material available to the golem race without destructible walls is just enough to make one shell.  Having the walls provide material would allow for a few shells to be produced.  I wouldn't consider this to be _that significant._  There are several other golem forms that are more prone to cause damage/chaos.  Uraninum and plasma forms come to mind and are plentiful to make.  

What's interesting about alien golems is that they might be suited towards fighting megafauna.  The speed and regen abilities will be very helpful.  I have never done this myself, but when I have fought megafauna in other golem forms, I was slaughtered pretty easily.

**Comment by iskyp** [comment]
nothing on lavaland except the necropolis should be indestructible

**Comment by Kromgar** [comment]
@iskyp what about clown ruin

**Comment by Anonmare** [comment]
>Alloy golems are the OP type
>Not plastitanium for being lava immune
>Not cloth golems for being able to revive
>Not sand golems for their massive brute resistance
>Not runic golems for their Dominate and short-range jaunt

**Comment by AlexTheSysop42** [comment]
>Not clockwork for gottagofast

**Comment by Repukan** [comment]
@Anonmare

Yes, and all of those need to be mined for, which is affected by rng and position of ship spawn. Runic Golems suffer from the same "Great I saw the path as a ghost so now I can beeline there to be stronger."

Clockwork golems aren't even constructable without admin fuckery. I don't understand why you're comparing the different golem species when a couple of them require two minerals at once and other two suffer from the same beeline problem. At least, for all it's faults, the alien alloy should be first gated behind them doing research in order to unlock it, instead of being able to be no slowdown passive healing high brute chance spaceproof marauders.

File: _maps/RandomRuins/LavaRuins/lavaland_surface_ufo_crash.dmm [hunk 1]
```diff
@@ -6,37 +6,29 @@
 /turf/open/floor/plating/asteroid/basalt/lava_land_surface,
 /area/lavaland/surface/outdoors)
 "c" = (
-/turf/closed/indestructible/abductor{
+/turf/closed/wall/mineral/abductor{
+	icon = 'icons/turf/walls.dmi';
 	icon_state = "alien20"
```
Comment (ninjanomnom) [inline _maps/RandomRuins/LavaRuins/lavaland_surface_ufo_crash.dmm]: the icon var, and for that matter icon_state as well should not be edited in maps, make a type which sets these properly

**Review by ninjanomnom — CHANGES_REQUESTED** [review]

**Comment by Barhandar** [comment]
>not clockwork servant for ARMOR WEARING GOLEM

@Repukan 
Alloy golems require surgery to actually be usable, due to the whole "superlingual matrix" thing. Giving them a little more alloy doesn't actually change anything.

File: _maps/RandomRuins/LavaRuins/lavaland_surface_ufo_crash.dmm (hunk 1 above)
Comment (ninjanomnom) [reply _maps/RandomRuins/LavaRuins/lavaland_surface_ufo_crash.dmm]: You're making it not indestructible so use the normal icons and remove these map edits

**Comment by AlexTheSysop42** [comment]
> > not clockwork servant for ARMOR WEARING GOLEM

That actually was op and was removed from Pride's Mirror. RIP coolest golem.

File: _maps/RandomRuins/LavaRuins/lavaland_surface_ufo_crash.dmm [hunk 2]
```diff
@@ -6,37 +6,29 @@
 /turf/open/floor/plating/asteroid/basalt/lava_land_surface,
 /area/lavaland/surface/outdoors)
 "c" = (
-/turf/closed/indestructible/abductor{
+/turf/closed/wall/mineral/abductor{
+	icon = 'icons/turf/walls.dmi';
```
Comment (ShizCalev) [inline _maps/RandomRuins/LavaRuins/lavaland_surface_ufo_crash.dmm]: subtype please.

**Review by ShizCalev — CHANGES_REQUESTED** [review]

File: _maps/RandomRuins/LavaRuins/lavaland_surface_ufo_crash.dmm (hunk 1 above)
Comment (ninjanomnom) [reply _maps/RandomRuins/LavaRuins/lavaland_surface_ufo_crash.dmm]: If you don't like the base icons then a subtype with the other icons would also be fine, you may be forced to convert it into a smoothed icon set though.

**Comment by iskyp** [comment]
wow i actually didn't expect that smoothing to make that look really good

**Review by SpaceManiac — APPROVED** [review]
The basalt layered under the walls alone makes this look better than it did before.

**Pull request merged**